.github/workflows/ci_cron_weekly.yml

```diff
@@ -67,7 +67,7 @@ jobs:
     # we include them just in the weekly cron. These also serve as a test
     # of using system libraries and using pytest directly.
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     name: More architectures
```

Comment:
Hmm the aarch64 job doesn't need emulator anymore but it is out of scope here. See https://github.com/astropy/photutils/pull/1997